### ISSUE TITLE
Missing a parentheses?

### ISSUE THREAD

**Issue by mikisama** [azure-rtos/threadx#215]
Hi,

The comments in the assembly code seem to be missing a parentheses.
Not only the risc-v32 iar port.

```diff
diff --git a/ports/risc-v32/iar/src/tx_thread_context_restore.s b/ports/risc-v32/iar/src/tx_thread_context_restore.s
index 8a23b6e..3d497f2 100644
--- a/ports/risc-v32/iar/src/tx_thread_context_restore.s
+++ b/ports/risc-v32/iar/src/tx_thread_context_restore.s
@@ -180,7 +180,7 @@ _tx_thread_context_restore:
     /* }  */
 _tx_thread_not_nested_restore:
     /* Determine if a thread was interrupted and no preemption is required.  */
-    /* else if (((_tx_thread_current_ptr) && (_tx_thread_current_ptr == _tx_thread_execute_ptr) 
+    /* else if (((_tx_thread_current_ptr) && (_tx_thread_current_ptr == _tx_thread_execute_ptr))
                || (_tx_thread_preempt_disable))
```

![image](https://user-images.githubusercontent.com/41532794/197453756-89837758-07ae-4011-9424-bbbd9a012f7b.png)

**Comment by goldscott** [comment]
Hi @mikisama - thanks for finding this! I'll close this ticket once we have a release that fixes these comments.

**Comment by TiejunMS** [comment]
Fixed in 6.2.1 release. Closing.
